### PR TITLE
Ctl ltl token 1997234

### PR DESCRIPTION
**Pull request by LenaSaid**
Checks the bounds to decide the icon of the verification result dialog, as well as informs the user if there were not enough tokens explored

Solves https://bugs.launchpad.net/tapaal/+bug/1997234.

### CODE REVIEW

**Comment by srba** [comment]
Does not seem to work for CTL in the bug report - open the first query and decrease the number of extra tokens and verify. It will answer FALSE (in red) which is wrong, it should be inconclusive (in yellow).

**Comment by LenaSaid** [comment]
> Does not seem to work for CTL in the bug report - open the first query and decrease the number of extra tokens and verify. It will answer FALSE (in red) which is wrong, it should be inconclusive (in yellow).

Should be shown in yellow now

File: src/main/java/dk/aau/cs/model/tapn/TAPNQuery.java [hunk 1]
```diff
@@ -27,9 +24,11 @@ public int getExtraTokens() {
...
 	public QueryType queryType(){
-		if(property instanceof TCTLEFNode) return QueryType.EF;
-		else if(property instanceof TCTLEGNode) return QueryType.EG;
-		else if(property instanceof TCTLAFNode) return QueryType.AF;
+		if (property instanceof TCTLEFNode) return QueryType.EF;
+		else if (property instanceof TCTLEGNode) return QueryType.EG;
+		else if (property instanceof TCTLAFNode) return QueryType.AF;
+		else if (property instanceof LTLENode) return  QueryType.E;
+		else if (property instanceof LTLANode) return QueryType.A;
 		else return QueryType.AG;
```
Comment (srba) [inline src/main/java/dk/aau/cs/model/tapn/TAPNQuery.java:32]: This looks weird, I would suggest to add a test for AG and in "else" return unknown.

Comment (LenaSaid) [reply src/main/java/dk/aau/cs/model/tapn/TAPNQuery.java]: Should be fixed now

**Pull request merged**